action/src/test/java/com/adyen/checkout/action/TestActionDelegate.kt

```diff
@@ -38,7 +38,7 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.flow.MutableStateFlow
-import java.util.Locale
+import java.util.*
```

Comment:
Is this your IDE messing up?

Reply:
Yes, apparently some setting was not right